wolfssl/wolfcrypt/sha256.h

```diff
@@ -96,6 +104,7 @@ typedef struct wc_Sha256 {
     word32  loLen;     /* length in bytes   */
     word32  hiLen;     /* length in bytes   */
     void*   heap;
+    const byte* data;
```

Comment:
Is this pointer needed in non intel asm cases?  I don't like to add size to these structures unless we have to.

Reply:
Removed